modules/serving-transport-encryption-trust-configuration.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/serving-transport-encryption.adoc
+:_mod-docs-content-type: CONCEPT
+[id="serving-transport-encryption-trust-configuration_{context}"]
+= Trust configuration
+
+When you enable any of the transport encryption features, you must make sure that all clients calling trust the Certificate Authority (CA) issuing the certificates used for the transport encryption.
+
+There are multiple places where trust must be ensured:
+
+* Cluster external client, such as Browser or other application. This is out of the scope of {ServerlessProductName}.
+
+* {ServerlessProductName} system components, such as Activator, Queue-Proxy, and Ingress-Controller.
+
+* Cluster internal client, such as a Knative Service or other workload.
+
+
+[id="serving-transport-encryption-trust-configuration-serving-and-services_{context}"]
+== Trust configuration for OpenShift Serverless Serving components and Knative Services
+
+To ensure that {ServerlessProductName} Serving components and Knative Services trust the CA that issues certificates, you can create a ConfigMap in the following namespaces with the label `networking.knative.dev/trust-bundle: true`:
++
```

Comment:
This `+` is rendered in plaintext in [the preview](https://82435--ocpdocs-pr.netlify.app/openshift-serverless/latest/knative-serving/serving-transport-encryption.html#serving-transport-encryption-trust-configuration-serving-and-services_serving-transport-encryption).